### ISSUE TITLE
Docksal for Windows Hyper-V error message

### ISSUE THREAD

**Issue by lpeabody** [docksal/docksal#357]
**Description**

I'm trying to install Docksal on Windows. Babun and Docksal installed cleanly but I get the following error message when I try to create the docksal machine.

**Steps to reproduce the issue:**

1. Install Babun.
2. `curl -fsSL get.docksal.io | sh`
3. `fin vm start`

**Describe the results you received:**

```
{ ~ }  » fin vm start
Creating docker machine 'docksal'...
Running pre-create checks...
Error with pre-create check: "This computer is running Hyper-V. VirtualBox won't boot a 64bits VM when Hyper-V is activated. Either use Hyper-V as a driver, or disable the Hyper-V hypervisor. (To skip this check, use --virtualbox-no-vtx-check)"
```

**Describe the results you expected:**

Expected the virtual machine to be created and start.

**Comment by lmakarov** [comment]
Are you trying to use Docker for Windows (uses Hyper-V) with Docksal or trying to use the standard VirtualBox setup?

If it's the former - follow the instructions [here](http://docs.docksal.io/en/master/getting-started/env-setup-native/)

If it's the latter, then the "answer in in the question":

"This computer is running Hyper-V. VirtualBox won't boot a 64bits VM when Hyper-V is activated. Either use Hyper-V as a driver, or disable the Hyper-V hypervisor. (To skip this check, use --virtualbox-no-vtx-check)"

In other worlds - VirtualBox and Hyper-V don't get along. Uninstall Hyper-V to get VirtualBox working again.

**Comment by lpeabody** [comment]
So I tried switching to VirtualBox away from Docker for Windows (because performance), and before doing so I disabled the Hyper-V feature in Windows. Problem is, when firing up Babun I all of a sudden can't do anything in it, complaining about how it can't fork a process or something (I should have copied the error message). In short Babun could not run any commands.

I enabled Hyper-V again and rebooted. Fired up Babun and all of a sudden I could issue commands normally again. So, Babun seems to have a dependency on Hyper-V possibly. Because of that, your instructions for using Docksal with Windows are essentially restricted to using Docker for Windows as the solution. The restriction comes into play because in your instructions you specifically call out using Babun to setup Docksal, which in my case can only be operated normally with Hyper-V enabled.

Anyway, I thought that was interesting and worth mentioning. Maybe something has changed since the instructions were last updated for Windows?
